app/src/features/city/subsectorSlice.ts

```diff
@@ -1,8 +1,8 @@
 import { RootState } from "@/lib/store";
 import { SubSectorAttributes } from "@/models/SubSector";
 import { PayloadAction, createSlice } from "@reduxjs/toolkit";
-import stationaryEnergyForm from './data/stationary-form-inputs.json'
 import { SubCategory, SubCategoryAttributes } from "@/models/SubCategory";
+import { SectorFormSchema } from "@/util/form-schema";
```

Comment:
```suggestion
import { type SectorFormSchema } from "@/util/form-schema";
```

Reply:
Same for e.g. SubCatergoryAttributes, as it is only used as a type definition and doesn't need to be part of the compiled JS output.